bode/bode/resources/tasks/api.py

```diff
@@ -17,3 +17,8 @@ def get(self):
     @blueprint.response(201, TaskSchema)
     def post(self, task_data):
         return Task.create(**task_data)
+    
+    @blueprint.arguments(TaskSchema)
```

Comment:
It's pretty standard to have actions regarding single resource grouped under `/resource/:id` path, in our case `/tasks/:task_id`. Then we don't need any input arguments, the `task_id` is enough information to delete a task. [Here you can find an example of this kind of setup](https://flask-smorest.readthedocs.io/en/latest/quickstart.html#simple-example). It's also a good practice to return the deleted resource in the response. We should also handle the case when someone provides a non-existing id and return `404 not found` status.